First batch of adding descriptions to the documentation (#14825)
netdata/netdata: docs/cloud/manage/sign-in.md

# Sign in to Netdata

> This page will guide you through on how to sign in to Netdata with email, Google or GitHub, plus provide some tips if you're having trouble signing in.

Reviewer: ```suggestion
This page explains how to sign in to Netdata with your email, Google account, or GitHub account, and provides some tips if you're having trouble signing in.
```
